examples/arkode/C_serial/CMakeLists.txt

```
@@ -22,6 +22,9 @@
 set(ARKODE_examples
     # tests that are always run
     "ark_analytic\;\;"
+    "lsrk_ssp_analytic\;\;"
+    "lsrk_analytic\;\;"
+    "lsrk_analytic_varjac\;\;"
```

Update with renaming suggestions
```suggestion
    "ark_analytic_ssprk\;\;"
    "ark_analytic_lsrk\;\;"
    "ark_analytic_varjac_lsrk\;\;"
```